O/ONNXRuntime/build_tarballs.jl

```diff
@@ -90,23 +101,22 @@ function platform_exclude_filter(p::Platform)
     Sys.isfreebsd(p)
 end
 platforms = supported_platforms(; exclude=platform_exclude_filter)
-platforms = expand_cxxstring_abis(platforms; skip=!Sys.islinux)
 
-cuda_platforms = Platform[]
-for cuda_version in cuda_versions
-    cuda_tag = "$(cuda_version.major).$(cuda_version.minor)"
-    for p in [
-        Platform("x86_64", "Linux"; cuda = cuda_tag),
-        Platform("x86_64", "Windows"; cuda = cuda_tag)
-    ]
-        push!(platforms, p)
-        push!(cuda_platforms, p)
+let cuda_platforms = CUDA.supported_platforms(min_version=v"10.2", max_version=v"11")
+    filter!(p -> !(arch(p) == "x86_64" && Sys.islinux(p) && p["cuda"] == "10.2"), cuda_platforms) # Fails with: nvcc error   : 'ptxas' died due to signal 11 (Invalid memory reference)
+    push!(cuda_platforms, Platform("x86_64", "Linux"; cuda = "11.3"))
+    push!(cuda_platforms, Platform("x86_64", "Windows"; cuda = "11.3"))
+
+     # Tag non-CUDA platforms matching CUDA platforms with cuda="none"
+    for platform in platforms
+        if CUDA.is_supported(platform) && arch(platform) != "powerpc64le" || platform == Platform("x86_64", "Windows")
```

Comment:
Why those additional checks? That's supposed to be handled by `CUDA.is_supported`: https://github.com/JuliaPackaging/Yggdrasil/blob/6feb2ef0524c363cdacd2da97be459a0ee74e702/platforms/cuda.jl#L204

Reply:
I want to _exclude_ PowerPC64LE, and include Windows.